### ISSUE TITLE
MaP Metric Keras Implementation [Motion]

### ISSUE THREAD

**Issue by JackWetherell** [waymo-research/waymo-open-dataset#324]
Is it possible to use the provided metrics (in particular the MaP metric) in the `waymo-open-dataset` library with keras (tf v2.0+)? 

How would I use the MaP as a training metric/loss in a keras model? (The MaP seems to be 0.0 in the tutorial example, and also this is using tf v1 and not used as the training loss itself.)

Many thanks!

**Comment by yuningchai** [comment]
Hi there,

I am not too familiar with tf2.0. Maybe somebody else can help you there.

The metric is not differentiable, so you won't be able to use it as a loss.

**Comment by JackWetherell** [comment]
Thank you for the information.
